### PR TITLE
virtme-init: Add default secure_path

### PR DESCRIPTION
**Pull request by marcosps**
When using vng without the --no-virtme-ng-init, running a command that resides on /sbin with sudo doesn't work:

$ vng --no-virtme-me-init
$ mpdesouza@virtme-ng:~/git/linux> modprobe
Absolute path to 'modprobe' is '/usr/sbin/modprobe', so running it may require superuser privileges (eg. root). $ mpdesouza@virtme-ng:~/git/linux> sudo modprobe
sudo: modprobe: command not found

In essence, the path when running sudo is the same of the normal user. Fix this problem by creating a secure_path when running with sudo, which includes /sbin  and usr/sbin along with the common binary paths. With the fix applied, modprobe is found as expected:

$ vng --no-virtme-me-init
$ mpdesouza@virtme-ng:~/git/linux> modprobe
Absolute path to 'modprobe' is '/usr/sbin/modprobe', so running it may require superuser privileges (eg. root). $ mpdesouza@virtme-ng:~/git/linux> sudo modprobe
modprobe: ERROR: missing parameters. See -h.

### CODE REVIEW

**Comment by marcosps** [comment]
@arighi IIUC, the same issue also exists with virtme-ng-init, since it only created the user entries in the sudoers without specifying the secure_path. At least in my tests the problem also existed.

**Comment by arighi** [comment]
Yep, virtme-ng-init is doing the same. I haven't seen this issue, because Ubuntu `/sbin` and `/usr/sbin` are already in the `$PATH` also for non-root users.

Could you rephrase the commit message to something like the following (or similar):

"When executing vng as a non-privileged user without /sbin in their $PATH, attempting to run a command located in that directory with sudo will not succeed."

In this way the issue is more clear and then we can fix also in virtme-ng-init. Thanks for noticing this!

**Comment by marcosps** [comment]
@arighi Done! Thanks for improving the commit message :)

**Comment by arighi** [comment]
Thank you, applied!

**Comment by arighi** [comment]
@marcosps fixed also in virtme-ng-init https://github.com/arighi/virtme-ng-init/commit/607150cef0adf49c3cbf5b2840ed517f75cba17c
